components/job-orchestration/job_orchestration/scheduler/compress/compression_scheduler.py

```diff
@@ -49,6 +50,18 @@
 
 scheduled_jobs = {}
 
+recieved_signal = False
+
+
+def signal_handler(sig, frame):
+    global recieved_signal
+    recieved_signal = True
```

Comment:
```suggestion
def sigterm_handler(sig, frame):
    global received_sigterm
    received_sigterm = True
```

Since we're only handling SIGTERM I think it makes sense to give more specific names.